tasks/setup.py

```diff
@@ -26,6 +26,17 @@ def setup(
     """
     Setup a local environment, interactively by default
     """
+    # Ensure aws cli is installed
+    result = ctx.run("aws --version", warn=True, hide=True)
```

Comment:
You could use [shutil.which](https://docs.python.org/3.11/library/shutil.html#shutil.which) (works on Windows too) with `if not shutil.which("aws"):`

Reply:
Ah, thank you!!

Reply:
Updated

Reply:
Nice!